### PR TITLE
Fix UI and add Legacy ESX group support

### PR DESCRIPTION
**Pull request by nsinisterra**
Hi there!

Simple PR but here's the issue https://github.com/izio38/iZone/issues/7 's fix. Chromium, as usuals, depreciate the toElement property for onClick events. I've replaced it by the new `target` property.

Added to that, I saw that the permissions system is deprecated, is based on essentialmode super-legacy system

Changelog
- Rename `event.toElement` to `event.target``
- Rebuild proejct
- Added `USE_ESX_ADMIN_SYSTEM` and `ESX_GROUPS_ENABLED` for ESX legacy version support (Using group names and not ids)

Hope it helps!

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -1,9 +1,12 @@
 Config = {
     USE_ESSENTIALMODE_ADMIN_SYSTEM = false,
     ESSENTIALMODE_PERMISSION_LEVEL_REQUIRED = 4,
+
+    USE_ESX_ADMIN_SYSTEM = true,
+    ESX_GROUPS_ENABLED = {'superadmin', 'admin'},
     -- only if USE_ESSENTIALMODE_ADMIN_SYSTEM is set to false
     ADMINS = {"steam:110000104bd6695", "license:4b8630ed9745879150c3e82f40b82d731017c77b"},
-    CONTROL_TO_OPEN_PANEL = 38,
+    CONTROL_TO_OPEN_PANEL = 344,
```
Comment (nsinisterra) [inline config.lua:9]: Updated from E (38) to F11 (344) in order to avoid problems with interactions

File: server.lua [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+ESX = nil
+
+TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
 
 function isPlayerAuthorized(source)
-    if (Config.USE_ESSENTIALMODE_ADMIN_SYSTEM) then
+    if (Config.USE_ESX_ADMIN_SYSTEM) then
+        -- Fetch the user and check permissions
+        local xPlayer = ESX.GetPlayerFromId(source)
+        local plyGroup = xPlayer.getGroup()
```
Comment (nsinisterra) [inline server.lua:10]: Hmm, sometimes this returns `SCRIPT ERROR: @iZone/server.lua:10: attempt to index a nil value (local 'xPlayer')`. I'll check it

**Comment by ShadowOfVoid** [comment]
thanks mate ! will try it now

**Comment by ShadowOfVoid** [comment]
still not working i guess , when i press manage zones i get -> Uncaught TypeError: Cannot read property 'id' of null (@izone/ui/public/build/bundle.js:2952)

**Comment by nsinisterra** [comment]
@ShadowOfVoid to fix the problem you need update the following files:
- ui/public/build/bundle.js
- ui/public/build/bundle.css.map

In github those are hidden by default because are transpiled versions.

Anyway, I guess that i'll remove the ESX V1 Group compatibility in order to enable the merge without any problem, and later with more time, I'll open it as a different PR.

**Comment by IzioDev** [comment]
Thanks for the contribution !

**Pull request merged**